x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/TokenServiceTests.java

```diff
@@ -143,7 +143,7 @@ public void setupClient() {
         // tokens docs on a separate index), let's test the TokenService works in a mixed cluster with nodes with versions prior to these
         // developments
         if (randomBoolean()) {
-            oldNode = addAnotherDataNodeWithVersion(this.clusterService, randomFrom(Version.V_6_7_0, Version.V_7_0_0));
+            oldNode = addAnotherDataNodeWithVersion(this.clusterService,  Version.V_7_0_0);
```

Comment:
should this be a range like so?

```
            oldNode = addAnotherDataNodeWithVersion(this.clusterService, VersionUtils.randomVersionBetween(random(), Version.V_7_0_0, Version.V_7_1_0));
```

Reply:
Makes sense given the comment above, will update this too.